### DISCUSSION TITLE
Executing install script ends in git fetch error

### DISCUSSION THREAD

**Discussion by kailuersystel** [Homebrew/discussions#528] [Getting started]
I am unable to execute the install script.

git is install via Xcode. Do you have any hints please? I dont know, if this is important: there was already a Homebrew installation based on the old ruby script. I might have failed to completely remove it.

==> Downloading and installing Homebrew...
remote: Enumerating objects: 52, done.
remote: Counting objects: 100% (52/52), done.
remote: Compressing objects: 100% (52/52), done.
Failed during: git fetch --force origin, 28.37 MiB | 14.12 MiB/s

Update: The script also logs those errors:
error: index-pack died of signal 25595), 19.37 MiB | 12.91 MiB/s
fatal: index-pack failed

**Comment by gromgit** [comment]
> there was already a Homebrew installation based on the old ruby script. I might have failed to completely remove it.

If `type -p brew` still outputs `/usr/local/bin/brew`, first try `brew update-reset` instead of running the installer all over again.

**Reply by kailuersystel** [reply]
Thank you, but it outputs "brew not found"

**Reply by gromgit** [reply]
See if `/usr/local/bin/brew` actually exists. If it does, try `/usr/local/bin/brew update-reset`. If that doesn't work, run the installer script again and see if the error reappears.

And if you're running macOS High Sierra or earlier, note that Homebrew is no longer supported on your OS version.

**Reply by kailuersystel** [reply]
"No such file". The installer is causing the problem. I am using the current MacOS version.

**Reply by gromgit** [reply]
If you've tried running the installer again, and the error still appears, try running the uninstaller script first (see https://github.com/Homebrew/install/#uninstall-homebrew) before installing again.

**Reply by kailuersystel** [reply]
I tried and the uninstaller yields something like "brew" not found.
